### DISCUSSION TITLE
different scala version between targets

### DISCUSSION THREAD

**Discussion by mtslh** [scalameta/metals#2945] [Q&A]
Hello.
I am using ubuntu 20.04 LTS x86_64 and here are other specs:

```
Scala code runner version 2.13.6 -- Copyright 2002-2021, LAMP/EPFL and Lightbend, Inc.
```
```
java 11.0.10 2021-01-19 LTS (oracle Java)
Java(TM) SE Runtime Environment 18.9 (build 11.0.10+8-LTS-162)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.10+8-LTS-162, mixed mode)
```
```
sbt --version
sbt version in this project: 1.5.4
sbt script version: 1.5.4
```
and **sublime text** with **sublimelsp/LSP** and **scalameta/metals-sublime**

the problem is this :
while building by shift+f10 is get this message **_you are using scala 2.12.14 which is not yet supported in this version of metals please downgrade to scala version 2.12.12  for the moment until the new metal release._**
as you can see in the "http://127.0.0.1:5031/doctor" print out below I am using 2.13.
I don't know how to fix this issue.
thanks for you help.

![](https://imgbb.online/images/2021/07/09/Screenshot-2021-07-09-at-06-40-06-Metals-Doctorc72b3d6b73b9b2c0.png)

**Comment by tgodzik** [comment]
Thanks for the question! `slm-build` is the sbt build target and sbt 1.5.4 is using currently Scala 2.12.14. You should update to the newest Metals version, which supports that Scala version, or ignore the warning currently if you don't plan to work on your build definition.

**Reply by mtslh** [reply]
Hello. thanks for your reply.
So I should update metals?
Could you tell me how?

**Reply by ayoub-benali** [reply]
@mtslh you can run `LSP-metals: settings` in the command palette and update the default Metals version. 
Latest release versions are found here https://scalameta.org/metals/docs/
